pyproject.toml

```diff
@@ -755,7 +755,7 @@ grpc = [
   "grpcio>=1.15.0",
 ]
 hashicorp = [
-  "hvac>=0.10",
+  "hvac>=1.1.0",
```

Comment:
I'm not sure, but seems like this changes made by pre-commit, seems like changes in https://github.com/apache/airflow/pull/36532 made before we move into the `pyproject.toml`

I think better to move this changes in the separate PR